### DISCUSSION TITLE
Frontend Application

### DISCUSSION THREAD

**Discussion by shaiktahseen** [community/community#74737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello

I want to contribute an open source any one help me to find out the companies, startups.

I don't know how to find out the open source contributions.

Thanks

**Comment by TheEdaL06** [comment]
Hello! It's great to hear that you're interested in contributing to open source projects. Here are some steps and tips to help you get started:

1. **Explore GitHub**: Visit [github.com/explore](https://github.com/collections/choosing-projects) to see a curated list of projects. You can also find beginner-friendly issues recommended for you to get started.

2. **Search for Projects**: You can search for open source projects on GitHub by using the search bar at the top of the page. You can filter your search by programming language, spoken language, and date.

3. **Look for 'Good First Issues'**: Many repositories tag some issues as 'good first issues' to indicate that they are beginner-friendly.

4. **Follow Open Source Contributors**: You can follow active contributors to see what projects they are working on.

5. **Contribute to What You Use**: If there's an open-source tool or library that you use frequently, consider contributing to that project. You're already familiar with it, and you'll understand the value of your contribution.

As for finding companies and startups that contribute to open source, here are some of the top contributors:

- **Microsoft**: Microsoft is one of the most active contributors to open source projects on GitHub.
- **Google**: Google is another tech giant that actively contributes to open source.
- **Red Hat**: Despite having a smaller staff than Microsoft and Google, Red Hat developers are very active when contributing on GitHub.
- **Amazon Web Services (AWS)**: AWS is a significant contributor to open-source software, with its employees spearheading over 1,200 projects on GitHub.

Remember, open source is not just about code. You can contribute in many ways, including documentation, design, answering questions, and more. Happy coding! 😊

**Reply by shaiktahseen** [reply, marked as answer]
Oh nice 👍 i got some good information.

May I know who is this?

**Reply by mdRaja987** [reply]
mdRaja987/mdRajamdrajak043@gmail.com
